distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/opensearch/OpenSearchEndPoint.java

```diff
@@ -0,0 +1,73 @@
+package org.codice.ddf.itests.common.opensearch;
+
+import com.jayway.restassured.response.ValidatableResponse;
+
+import org.codice.ddf.itests.common.AbstractIntegrationTest;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+
+import static com.jayway.restassured.RestAssured.when;
+import static com.jayway.restassured.RestAssured.given;
+
+/**
+ * This class is used to perform open search query on a DDF end point
+ * that a client accesses to send query parameters and receive search results
+* */
+public class OpenSearchEndPoint {
+
+    public static final AbstractIntegrationTest.DynamicPort HTTPS_PORT =
+            new AbstractIntegrationTest.DynamicPort("org.codice.ddf.system.httpsPort", 2);
+
+    public static final AbstractIntegrationTest.DynamicUrl SERVICE_ROOT =
+            new AbstractIntegrationTest.DynamicUrl(AbstractIntegrationTest.DynamicUrl.SECURE_ROOT,
+                    HTTPS_PORT,
+                    "/services");
+
+    public static final AbstractIntegrationTest.DynamicUrl REST_PATH =
+            new AbstractIntegrationTest.DynamicUrl(SERVICE_ROOT, "/catalog/");
+
+    public static final AbstractIntegrationTest.DynamicUrl OPENSEARCH_PATH =
+            new AbstractIntegrationTest.DynamicUrl(REST_PATH, "query");
+
+    protected static final Logger LOGGER = LoggerFactory.getLogger(AbstractIntegrationTest.class);
+
+    /**
+     * Gets the validatble response of the open search query
+     * When admin is set to true, there is additional authentication specifications
+     * for setting up as an admin
+     *
+     * @params format       The search format to be used. Default is atom.
+     * @params admin        If admin is true, search is for Admin auth
+     * @params queryParams  Input search query parameters to create an OpenSearch query
+     *
+     * @return  Returns the ValidatableResponse object that will be used for validation
+    * */
+    public static ValidatableResponse getOpenSearch(String format, boolean admin,
```

Comment:
❓ Instead of using an `admin` boolean flag, should we just have a different method called `getOpenSearchAsAdmin`, or one that takes in username and a password as extra arguments? Boolean are always confusing and have to be looked up to know what they actually mean.